spec/knapsack/adapters/rspec_adapter_spec.rb

```diff
@@ -24,19 +24,19 @@
       end
 
       it do
+        expect(config).to receive(:prepend_before).with(:context).and_yield
         expect(config).to receive(:prepend_before).with(:each).and_yield
-        expect(config).to receive(:append_after).with(:each).and_yield
+        expect(config).to receive(:append_after).with(:context).and_yield
         expect(config).to receive(:after).with(:suite).and_yield
         expect(::RSpec).to receive(:configure).and_yield(config)
 
         expect(::RSpec).to receive(:current_example).twice.and_return(current_example)
         expect(described_class).to receive(:test_path).with(example_group).and_return(test_path)
 
         allow(Knapsack).to receive(:tracker).and_return(tracker)
-        expect(tracker).to receive(:test_path=).with(test_path)
-        expect(tracker).to receive(:start_timer)
-
-        expect(tracker).to receive(:stop_timer)
+        expect(tracker).to receive(:start_timer).ordered
```

Comment:
Nice improvement with `ordered`. I did not know about it 
https://relishapp.com/rspec/rspec-mocks/docs/setting-constraints/message-order